HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -465,6 +466,13 @@
     </sch:rule>
   </sch:pattern>
 
+  <sch:pattern>
+    <sch:title>[WindowStormWindow]</sch:title>
+    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Enclosure/h:Windows/h:Window/h:StormWindow'>
+      <sch:assert role='ERROR' test='h:GlassType[text()="clear" or text()="low-e"]'>Expected GlassType to be 'clear' or 'low-e'</sch:assert>
```

Comment:
What you have might work fine, but I suggest being more consistent with our other Schematron checks, which have stood the test of time:
```suggestion
      <sch:assert role='ERROR' test='count(h:GlassType) &lt;= 1'>Expected 0 or 1 element(s) for xpath: GlassType</sch:assert>
      <sch:assert role='ERROR' test='h:GlassType[text()="clear" or text()="low-e"] or not(h:GlassType)'>Expected GlassType to be 'clear' or 'low-e'</sch:assert>
```